src/inmanta/protocol/openapi/converter.py

```diff
@@ -94,9 +94,10 @@ def generate_openapi_definition(self) -> OpenAPI:
         for path, methods in self.global_url_map.items():
             api_methods = self._filter_api_methods(methods)
             if len(api_methods) > 0:
-                path_in_openapi_format = self._format_path(path)
```

Comment:
The `_format_path` method can probably go now.

Reply:
I thought I removed it but it seems I didn't. Thanks!